File: FastClass/salida/java/TODO.java
public class TODO { 

	public TODO() {
	}

	public class test1 extends Padre implements Interfaz, Interfaz2 { 
					String property;
	int read;
	Libro read_write;
	Cuaderno write_constructor;

		public test1(String property, Cuaderno write_constructor) {
			this.property = property;
			this.write_constructor = write_constructor;
		}
		public String getProperty(){
			return this.property;
		}
		public void setProperty(String property){
			this.property = property;
		}
		public int getRead(){
			return this.read;
		}
		public Libro getRead_write(){
			return this.read_write;
		}
		public void setRead_write(Libro read_write){
			this.read_write = read_write;
		}
		public void setWrite_constructor(Cuaderno write_constructor){
			this.write_constructor = write_constructor;
		}

		public class sub_clase1 extends Padre implements Interfaz, Interfaz2 { 
			String read_write;

			public sub_clase1() {
			}
			public String getRead_write(){
				return this.read_write;
			}
			public void setRead_write(String read_write){
				this.read_write = read_write;
			}

			public class sub_sub_clase_1 implements Interfaz, Interfaz2 { 
				String read_only;

				public sub_sub_clase_1() {
				}
				public String getRead_only(){
					return this.read_only;
				}

				public class sub_sub_sub_clase1 extends Padre implements Interfaz { 
					String write_only;

					public sub_sub_sub_clase1() {
					}
					public void setWrite_only(String write_only){
						this.write_only = write_only;
					}

				}
			}
		}
		public class sub_clase2 extends Padre { 
			String read_write;

			public sub_clase2(String read_write) {
				this.read_write = read_write;
			}
			public String getRead_write(){
				return this.read_write;
			}
			public void setRead_write(String read_write){
				this.read_write = read_write;
			}

		}
	}
	public class test2 extends Padre implements Interfaz, Interfaz2 { 
					String property;
	int read;
	Libro read_write;
	Cuaderno read_constructor;

		public test2(String property, Cuaderno read_constructor) {
			this.property = property;
			this.read_constructor = read_constructor;
		}
		public String getProperty(){
			return this.property;
		}
		public void setProperty(String property){
			this.property = property;
		}
		public int getRead(){
			return this.read;
		}
		public Libro getRead_write(){
			return this.read_write;
		}
		public void setRead_write(Libro read_write){
			this.read_write = read_write;
		}
		public void setRead_constructor(Cuaderno read_constructor){
			this.read_constructor = read_constructor;
		}

	}
	public class test3 { 

		public test3() {
		}

	}
	public class test4 { 
					String property;
	int read;
	int write;
	int const;

		public test4(String property, int const) {
			this.property = property;
			this.const = const;
		}
		public String getProperty(){
			return this.property;
		}
		public void setProperty(String property){
			this.property = property;
		}
		public int getRead(){
			return this.read;
		}
		public void setWrite(int write){
			this.write = write;
		}

	}
	public class test5 { 
					int readwrite;
	int writeconst;
	int readcont;
	int leescrcom;

		public test5(int writeconst, int readcont, int leescrcom) {
			this.writeconst = writeconst;
			this.readcont = readcont;
			this.leescrcom = leescrcom;
		}
		public int getReadwrite(){
			return this.readwrite;
		}
		public void setReadwrite(int readwrite){
			this.readwrite = readwrite;
		}
		public void setWriteconst(int writeconst){
			this.writeconst = writeconst;
		}
		public int getReadcont(){
			return this.readcont;
		}
		public int getLeescrcom(){
			return this.leescrcom;
		}
		public void setLeescrcom(int leescrcom){
			this.leescrcom = leescrcom;
		}

	}
	public class test6 extends Padre { 
					String property;
	int read;
	Libro read_write;
	Cuaderno write_constructor;

		public test6(String property, Cuaderno write_constructor) {
			this.property = property;
			this.write_constructor = write_constructor;
		}
		public String getProperty(){
			return this.property;
		}
		public void setProperty(String property){
			this.property = property;
		}
		public int getRead(){
			return this.read;
		}
		public Libro getRead_write(){
			return this.read_write;
		}
		public void setRead_write(Libro read_write){
			this.read_write = read_write;
		}
		public void setWrite_constructor(Cuaderno write_constructor){
			this.write_constructor = write_constructor;
		}

	}
	public class test7 extends Dad implements Interfaz { 
		String p;

		public test7(String p) {
			this.p = p;
		}
		public String getP(){
			return this.p;
		}
		public void setP(String p){
			this.p = p;
		}

	}
}

File: FastClass/salida/php/TODO.php
<?php 
class TODO { 

	public __construct() {
	}

	class test1 extends Padre implements Interfaz, Interfaz2 { 
					public $property;
	public $read;
	public $read_write;
	public $write_constructor;

		public __construct($property, $write_constructor) {
			$this->property = $property;
			$this->write_constructor = $write_constructor;
		}
		public function getProperty(){
			return $this->property;
		}
		public function setProperty($property){
			$this->property = $property;
		}
		public function getRead(){
			return $this->read;
		}
		public function getRead_write(){
			return $this->read_write;
		}
		public function setRead_write($read_write){
			$this->read_write = $read_write;
		}
		public function setWrite_constructor($write_constructor){
			$this->write_constructor = $write_constructor;
		}

		class sub_clase1 extends Padre implements Interfaz, Interfaz2 { 
			public $read_write;

			public __construct() {
			}
			public function getRead_write(){
				return $this->read_write;
			}
			public function setRead_write($read_write){
				$this->read_write = $read_write;
			}

			class sub_sub_clase_1 implements Interfaz, Interfaz2 { 
				public $read_only;

				public __construct() {
				}
				public function getRead_only(){
					return $this->read_only;
				}

				class sub_sub_sub_clase1 extends Padre implements Interfaz { 
					public $write_only;

					public __construct() {
					}
					public function setWrite_only($write_only){
						$this->write_only = $write_only;
					}

				}
			}
		}
		class sub_clase2 extends Padre { 
			public $read_write;

			public __construct($read_write) {
				$this->read_write = $read_write;
			}
			public function getRead_write(){
				return $this->read_write;
			}
			public function setRead_write($read_write){
				$this->read_write = $read_write;
			}

		}
	}
	class test2 extends Padre implements Interfaz, Interfaz2 { 
					public $property;
	public $read;
	public $read_write;
	public $read_constructor;

		public __construct($property, $read_constructor) {
			$this->property = $property;
			$this->read_constructor = $read_constructor;
		}
		public function getProperty(){
			return $this->property;
		}
		public function setProperty($property){
			$this->property = $property;
		}
		public function getRead(){
			return $this->read;
		}
		public function getRead_write(){
			return $this->read_write;
		}
		public function setRead_write($read_write){
			$this->read_write = $read_write;
		}
		public function setRead_constructor($read_constructor){
			$this->read_constructor = $read_constructor;
		}

	}
	class test3 { 

		public __construct() {
		}

	}
	class test4 { 
					public $property;
	public $read;
	public $write;
	public $const;

		public __construct($property, $const) {
			$this->property = $property;
			$this->const = $const;
		}
		public function getProperty(){
			return $this->property;
		}
		public function setProperty($property){
			$this->property = $property;
		}
		public function getRead(){
			return $this->read;
		}
		public function setWrite($write){
			$this->write = $write;
		}

	}
	class test5 { 
					public $readwrite;
	public $writeconst;
	public $readcont;
	public $leescrcom;

		public __construct($writeconst, $readcont, $leescrcom) {
			$this->writeconst = $writeconst;
			$this->readcont = $readcont;
			$this->leescrcom = $leescrcom;
		}
		public function getReadwrite(){
			return $this->readwrite;
		}
		public function setReadwrite($readwrite){
			$this->readwrite = $readwrite;
		}
		public function setWriteconst($writeconst){
			$this->writeconst = $writeconst;
		}
		public function getReadcont(){
			return $this->readcont;
		}
		public function getLeescrcom(){
			return $this->leescrcom;
		}
		public function setLeescrcom($leescrcom){
			$this->leescrcom = $leescrcom;
		}

	}
	class test6 extends Padre { 
					public $property;
	public $read;
	public $read_write;
	public $write_constructor;

		public __construct($property, $write_constructor) {
			$this->property = $property;
			$this->write_constructor = $write_constructor;
		}
		public function getProperty(){
			return $this->property;
		}
		public function setProperty($property){
			$this->property = $property;
		}
		public function getRead(){
			return $this->read;
		}
		public function getRead_write(){
			return $this->read_write;
		}
		public function setRead_write($read_write){
			$this->read_write = $read_write;
		}
		public function setWrite_constructor($write_constructor){
			$this->write_constructor = $write_constructor;
		}

	}
	class test7 extends Dad implements Interfaz { 
		public $p;

		public __construct($p) {
			$this->p = $p;
		}
		public function getP(){
			return $this->p;
		}
		public function setP($p){
			$this->p = $p;
		}

	}
}
?>
